plugins/inventory/now.py

```diff
@@ -418,12 +418,21 @@ def _merge_instance_config(self, instance_config, instance_env):
         return instance
 
     def _get_instance_from_env(self):
+        client_secret = os.getenv("SN_SECRET_ID")
+        if client_secret:
+            # Remove this in 3.0.0
+            self.display.deprecated("Setting environment variable 'SN_SECRET_ID' is being removed "
+                                    "in favor of 'SN_CLIENT_SECRET'",
+                                    version='3.0.0', collection_name='servicenow.itsm')
+        else:
+            client_secret = os.getenv("SN_CLIENT_SECRET")
```

Comment:
If both environment variables are defined, then the deprecated one wins. Perhaps we should favor the new one? Or explicitly allow only one to be defined, so that we further discourage the use of the deprecated one in new inventory files?